bss/bindings/sro/state_root_oracle.go

```diff
@@ -0,0 +1,616 @@
+// Code generated - DO NOT EDIT.
```

Comment:
Might be a naive Go question, but should this be checked-in, or generated by the build?

Reply:
Generally we'd like to have these checked in.

Reply:
Generated Go code should be checked in. We can use the `go:generate` thing to automate re-generating it, and we should add something to CI to re-generate everything and diff with the existing files, to make sure the generated code is correct.

Reply:
FYI a lot of these are generated with `make` (easier to handle with the output of `hardhat`), but adding a CI step to check that the generated files are up to date would be a nice to have.